ingest/v2.md

```diff
@@ -0,0 +1,101 @@
+| `type`       | string | The type of signal. While it is not enforced, we recommend structuring your signal names in namespaces separated by dots, with the signal type beginning with a lower case letter and any namespaced beginning with an uppercase letter. |
+
+The following properties are optional:
+
+| Property     | Type   | Description                                                                                 |
+| ------------ | ------ | ------------------------------------------------------------------------------------------- |
+| `sessionID`  | string | The user's session ID. This should be the same value for the same session/user combination. |
+| `payload`    | object | A JSON object with additional data.                                                         |
+| `isTestMode` | bool   | If `true`, the signal will be excluded from production queries. Defaults to `false`.        |
+
+## Payload
+
+The `payload` object can contain data you want to send to TelemetryDeck. It must be a **JSON object containing primitives**. Here's an example:
+
+```json
+{
+  "appID": "AAAA-BBBBBBBB-CCCC-DDDD",
+  "clientUser": "myClientUserHash",
+  "type": "Curl.Development.apiCall",
+  "sessionID": "mySessionID",
+  "isTestMode": false,
+  "payload": {
+    "TelemetryDeck.RunContext.locale": "en_US",
+    "TelemetryDeck.Device.architecture": "x86_64",
+    "floatValue": 3.14159
+  }
+}
+```
+
+Payload keys should be in the format `Namespace.OptionalSubNamespace.key`, with the namespaces capitalized and the key starting with a lowercase letter. This is not enforced but helps you organize your payload dimensions and fit in better with TelemetryDeck's internal payload keys.
+
+The value can be any primitive JSON value, but we'll convert it to a string before storing it, except for a number of special cases that are stored as their original type.
+
+The payload should not contain nested objects. The behavior of arrays of strings is left undefined and should be avoided (except if you have a deep knowledge of [multi-value dimensions in Apache Druid](https://druid.apache.org/docs/latest/querying/multi-value-dimensions/)).
+
+## Special Payload Values
+
+The following payload values are not converted to strings, but need to be sent as their respective types:
+
+| Property     | Type  | Description                                                                                             |
+| ------------ | ----- | ------------------------------------------------------------------------------------------------------- |
+| `floatValue` | Float | A floating point number. Use for any numeric operations, such as building averages or adding up values. |
+
+{% noteinfo "More special values" %}
+More special values are coming soon for internal use.
+
+- The ones being defined in [this PR](https://github.com/TelemetryDeck/docs/pull/85) need to be implemented server side first.
```

Comment:
Rather than linking to the PR, we should replace this with the link to the doc, I think. But I can do that inside the PR itself so when it gets merged this article automatically gets updated, so also not a blocker.

Reply:
Fantastic, please do that when you merge.